changelogs/fragments/api.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - use get_record_by_sys_id instead of get_record in methods update, delete
```

Comment:
```suggestion
  - use get_record_by_sys_id instead of get_record in methods update, delete (https://github.com/ansible-collections/servicenow.itsm/pull/307).
```